conftest.py

```diff
@@ -1,32 +1,22 @@
 """
 Various common utilities for testing.
 """
-import contextlib
-import functools
-import json
-import multiprocessing
-import os
 import pathlib
-import queue
-import re
-import shutil
 import sys
-import tempfile
-import textwrap
-import time
 
-import pexpect
 import pytest
 
+from test_fixture.decorator import *  # noqa: F401, F403
+from test_fixture.fixture import *  # noqa: F401, F403
```

Comment:
I'm not very keen on this. I think it would be better to a) avoid * imports, b) import things from the module where they are defined rather than conftest.py which is supposed to be a pytest-specific module not something we import from. Though we had started doing that a lot.

Reply:
I made decorators directly imported from `pyodide-test-runner`. Note that it is not a proper package yet, instead I injected import path in `conftest.py`.